project-base/storefront/components/Pages/Search/SearchProducts.tsx

```diff
@@ -81,7 +81,7 @@ export const SearchProducts: FC = () => {
                             sorting={searchProductsData.orderingMode}
                             totalCount={searchProductsData.totalCount}
                             customSortOptions={[
-                                TypeProductOrderingModeEnum.Priority,
+                                TypeProductOrderingModeEnum.Relevance,
```

Comment:
This change should be mentioned in the upgrade notes. (for storefront) 

use `./phing upgrade-generate` to generate templates and take a look at the other upgrade notes for inspiration